dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task53800ChangeContainerPathToAbsoluteTest.java

```diff
@@ -0,0 +1,462 @@
+
+        task53800ChangeContainerPathToAbsolute.executeUpgrade();
+
+        checkTemplateFromDataBase(host, template);
+    }
+
+    /**
+     * Method to Test: {@link Task53800ChangeContainerPathToAbsolute#executeUpgrade()}
+     * When: Exists A legacy HTML TemplateLayout with relative path container
+     * Should: Should change the relative path by absolute path
+     */
+    @Test
+    public void whenLegacyTemplateLayoutHasRelativePathShouldTurnIntoAbsolutePath() throws DotDataException {
+        final Host host = new SiteDataGen().nextPersisted();
+        final String layout = String.format(legacyHTMLLayout, "");
+
+        final Contentlet theme = new ThemeDataGen().nextPersisted();
+        final Template template = new TemplateDataGen()
+                .title("template_test_" + System.currentTimeMillis())
+                .theme(theme)
+                .drawedBody(layout)
+                .body(String.format(body, ""))
+                .host(host)
+                .nextPersisted();
+
+
+        final Task53800ChangeContainerPathToAbsolute task53800ChangeContainerPathToAbsolute =
+                new Task53800ChangeContainerPathToAbsolute();
+
+        task53800ChangeContainerPathToAbsolute.executeUpgrade();
+        CacheLocator.getTemplateCache().remove(template.getInode());
+
+        checkTemplateFromDataBase(host, template);
+    }
+
+    /**
+     * Method to Test: {@link Task53800ChangeContainerPathToAbsolute#executeUpgrade()}
+     * When: Exists A legacy HTML TemplateLayout with absolute path container
+     * Should: Should keep the absolute path
+     */
+    @Test
+    public void whenLegacyTemplateLayoutHasAbsolutePath() throws DotDataException {
+        final Host host = new SiteDataGen().nextPersisted();
+        final String layout = String.format(legacyHTMLLayout, "//" + host.getHostname());
+
+        final Contentlet theme = new ThemeDataGen().nextPersisted();
+        final Template template = new TemplateDataGen()
+                .title("template_test_" + System.currentTimeMillis())
+                .theme(theme)
+                .drawedBody(layout)
+                .body(String.format(body, "//" + host.getHostname()))
+                .host(host)
+                .nextPersisted();
+
+
+        final Task53800ChangeContainerPathToAbsolute task53800ChangeContainerPathToAbsolute =
+                new Task53800ChangeContainerPathToAbsolute();
+
+        task53800ChangeContainerPathToAbsolute.executeUpgrade();
+        CacheLocator.getTemplateCache().remove(template.getInode());
+
+        checkTemplateFromDataBase(host, template);
+    }
+
+    /**
+     * Method to Test: {@link Task53800ChangeContainerPathToAbsolute#executeUpgrade()}
+     * When: Exists More than one template with relative path
+     * Should: Should change the relative path by absolute path
+     */
+    @Test
+    public void whenHaveMoreThanOneTemplate() throws IOException, DotDataException {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6034685)